bp_fe/syn/Makefile

```diff
@@ -3,27 +3,55 @@ TOP ?= $(shell git rev-parse --show-toplevel)
 
 include $(TOP)/Makefile.common
```

Comment:
Should be able to revert and run test with 

make build.v sim.v TB=uvm PROG=uvm_base_test

Reply:
This works, waiting on the other infrastructure changes with outstanding questions below to be complete before commiting.

Reply:
Change commited.